<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetallesOperacion</class>
 <widget class="QDialog" name="DetallesOperacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalles de Operación</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Información General</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelIdLabel">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelId"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFechaLabel">
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelFecha"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTipoLabel">
        <property name="text">
         <string>Tipo:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelTipo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelOrigenLabel">
        <property name="text">
         <string>Origen:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelOrigen"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDestinoLabel">
        <property name="text">
         <string>Destino:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelDestino"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelPlazaLabel">
        <property name="text">
         <string>Plaza:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelPlaza"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelTransportadoraLabel">
        <property name="text">
         <string>Transportadora:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelTransportadora"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelMonedaLabel">
        <property name="text">
         <string>Moneda:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="labelMoneda"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelMonedaDestinoLabel">
        <property name="text">
         <string>Moneda Destino:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="labelMonedaDestino"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Denominaciones</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableDenominaciones"/>
      </item>
      <item>
       <widget class="QLabel" name="labelTotal">
        <property name="text">
         <string>Total: $0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight</set>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DetallesOperacion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DetallesOperacion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>